specs/wp-invite-users-spec.js

```diff
@@ -94,11 +94,27 @@ testDescribe( `[${ host }] Invites:  (${ screenSize })`, function() {
 			} );
 
 			test.it( 'Sends an invite', function() {
-				return this.invitePeoplePage.inviteSent().then( sent => {
-					assert.equal( sent, true, 'The sent confirmation message was not displayed' );
+				this.invitePeoplePage.inviteSent().then( sent => {
+					return assert.equal( sent, true, 'The sent confirmation message was not displayed' );
 				} );
 			} );
 
+			test.it(
+				'Can see pending invite',
+				function() {
+					this.invitePeoplePage.backToPeopleMenu();
+
+					this.peoplePage = new PeoplePage( driver );
+					this.peoplePage.selectInvites();
+					return this.peoplePage.getMostRecentPendingInviteEmail().then( mostRecentPendingInviteEmail => {
+						return assert.equal(
+							mostRecentPendingInviteEmail,
+							newInviteEmailAddress,
+							'The email address prefilled on the accept invite page is not correct' );
```

Comment:
I'm not sure how this message is related to this test step; could you update it to point to the exact error that might be encountered here?